packages/usb/usb.1.3.1/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+name: "usb"
+version: "1.3.1"
```

Comment:
Name and version are redundant, opam will override them with what comes from the path. Can you remove them?

Reply:
I will remove the two lines in the next commit. Thank you!